samples/Zirku/Zirku.Client2/Program.cs

```diff
@@ -1,4 +1,10 @@
 var builder = WebApplication.CreateBuilder(args);
+
 var app = builder.Build();
+
 app.UseStaticFiles();
-app.Run();
\ No newline at end of file
+
+// Default redirect to the index page
+app.MapGet("/", () => Results.Redirect("/index.html"));
```

Comment:
That works but let's try to use `app.UseDefaultFiles()` instead (it must appear before the static files middleware): https://learn.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-8.0#serve-default-documents

Reply:
Yep, this is a better solution.